controllers/get.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("../models/get");
+
+exports.getTopics = (req, res) => {
+  fetchTopics().then((topics) => {
+    res.status(200).send({ topics });
+  });
```

Comment:
you should probably add a .catch block here just incase anything goes wrong, maybe also add a default 500 handler too :)